src/style.css

```diff
@@ -0,0 +1,120 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;0,1000;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900;1,1000&display=swap');
+
+* {
+  box-sizing: border-box;
+  font-family: 'Nunito', sans-serif;
+}
+
+body {
+  height: 100vh;
+  background: linear-gradient(to bottom, white, lightgray);
+}
+
+ul {
+  list-style: none;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+h2 {
+  text-align: center;
+  width: 100%;
+  margin-top: 10px;
+  font-weight: 300;
+  font-size: 32px;
+}
+
+img {
+  width: 70px;
+  background-color: white;
+  height: 70px;
+  border-radius: 7px;
+}
+
+.header-div {
+  margin-top: 60px;
+  justify-content: center;
+  display: flex;
+  align-items: center;
+}
+
+.list-wrapper {
+  width: 500px;
+  background: white;
+  box-shadow: 0 2px 3px rgba(255, 255, 255, 1);
+  margin-left: 200px;
+}
```

Comment:
![Screenshot from 2023-06-21 11-36-34](https://github.com/Samuelwanza/todolist_review/assets/100193711/e7b1ab7c-88c5-4d2f-a4bf-5ee83aa1b9a0)

It would be better if you Make you your list wrapper to align centrally other than having the default left margin of 200px as below
![Screenshot from 2023-06-21 11-43-45](https://github.com/Samuelwanza/todolist_review/assets/100193711/6b95ff1b-cf57-4a4e-9759-b1449ce703c8)